docs/types/reference-types.rst

```diff
@@ -160,6 +160,29 @@ always use one of the value types ``bytes1`` to ``bytes32`` because they are muc
     that you are accessing the low-level bytes of the UTF-8 representation,
     and not the individual characters.
 
+.. index:: ! bytes-concat
```

Comment:
The first paragraph of this section still sits `abi.encodePacked` for concatenation.

Reply:
Changed to `bytes.concat(bytes(s1), bytes(s2))`. Also added a test for this particular case as well. We didn't have the one with strings converted to bytes.